server/server.go

```diff
@@ -1,59 +1,70 @@
 package server
 
 import (
-    "log"
-    "fmt"
-    "net/http"
-    "os"
-    "database/sql"
+	"database/sql"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"reflect"
 )
 
+//TODO add license
+//TODO Godocs?
+//TODO unit tests
 var db *sql.DB // global variable to share it between main and the HTTP handler
```

Comment:
just noticed this now, but I don't think db can be accessed from main unless it's made to be DB